api/controllers/site.js

```diff
@@ -51,7 +53,7 @@ module.exports = {
 
     Promise.resolve(Number(req.params.id))
       .then((id) => {
-        if (isNaN(id)) {
+        if (_.isNaN(id)) {
```

Comment:
s this necessary? this style isNaN is not used throughout the codebase

Reply:
The `isNaN` was throwing linting error's so I used the underscore version.